### DISCUSSION TITLE
Help with major lagging

### DISCUSSION THREAD

**Discussion by jbatchler** [opensteno/plover#1649] [Q&A and Support]
I'm sorry. I'm not really that tech-savvy. I know some things, but coding is my father's business. I'm in court transcription and am wanting to learn steno privately, so I apologize if this seems way too easy to solve. I just haven't found an answer at all in this github discussion forum, and Nolltronics said you would be the people to talk to.

I'm attaching a video of my inputs. I hit EU fifty times and then the * fifty times. You can see my issue clearly. (It lags, and when I switch to the *, it's still filling in the EU inputs I made and then jumps. So when I'm typing, I can't see if my inputs are correct or not until it FINALLY catches up about twenty strokes later.) It renders my use of Plover completely ineffective. (Video link below.)

https://photos.app.goo.gl/if3YXLx4WxMqQDpH7

Another issue that pops up is that doing the same thing on other keys will make strange words that I didn't write.

It's been getting progressively worse to the point that I cannot use the keyboard anymore.

Any suggestions?

**Comment by aerickt** [comment]
I believe this may be an issue with Notepad specifically. Several people have reported similar issues (i.e. lagging output and words being jumbled) on Notepad in Windows 11. Have you tried writing output in another application like a browser or MS Word?

**Reply by jbatchler** [reply]
You've got to be kidding me. It worked. I feel like I just contacted tech support and they asked if my computer was turned off and on again.

But then here's my question, why was it progressively worse? It doesn't work for me on Chrome that well, but it says as much when you've downloaded the software. On Notepad, it was great, then okay, then poor, then like this. Why the progression?

**Reply by aerickt** [reply]
This is the first time I've heard of Notepad progressively getting worse; and to be honest, I'm not sure why Notepad is problematic in the first place. Some applications just don't respond well to how Plover emulates key events and Notepad seems to be [especially bad not just with Plover](https://github.com/openstenoproject/plover/issues/1619#issuecomment-1698414639).

Also, when Plover starts to act up a little, you _might_ be able to resolve the issue by just quitting and restarting Plover (not sure if this will work for your issue specifically). Closing the window itself isn't enough, you'll need to right click the icon in the bottom right and press "Quit Plover".

**Reply by jbatchler** [reply]
Thanks for the info. I honestly thought if Notepad is meant for bare text, why shouldn't it work? But lo, here we are.

I would turn Plover off (by right clicking on my taskbar), turn it back on, refresh my computer, reboot, everything, and it got progressively worse. I'll just know to avoid it in the future. Hopefully I won't encounter the same elsewhere.

Thanks again.

**Comment by gdwaynewarner** [comment]
Hi.

I was going to suggest "Wordpad" instead of Notepad, but that one appears
to be dead. Fortunately, you should be able to find something similar here
<https://alternativeto.net/software/wordpad>.   Good luck!
